cirq/protocols/__init__.py

```diff
@@ -54,6 +54,7 @@
 from cirq.protocols.inverse import (
     inverse,
 )
+from cirq.protocols.json import (to_json, read_json, to_json_dict)
```

Comment:
nit: follow others in file and one per line.

Reply:
I did! the formatter put it like this